pkg/cmd/template/data_values_flags.go

```diff
@@ -135,15 +137,15 @@ func (s *DataValuesFlags) AsOverlays(strict bool) ([]*datavalues.Envelope, []*da
 	return overlayValues, libraryOverlays, nil
 }
 
-func (s *DataValuesFlags) file(path string, strict bool) ([]*datavalues.Envelope, error) {
-	libRef, path, err := s.libraryRefAndKey(path)
+func (s *DataValuesFlags) file(fullPath string, strict bool) ([]*datavalues.Envelope, error) {
+	libRef, path, err := s.libraryRefAndKey(fullPath)
 	if err != nil {
 		return nil, err
 	}
 
 	contents, err := s.readFile(path)
 	if err != nil {
-		return nil, fmt.Errorf("Reading file '%s': %s", path, err)
+		return nil, fmt.Errorf("Unable to read file '%s' from flag value '%s': %s", path, err)
```

Comment:
There are only two arguments here so the error message is showing me "MISSING" after the colon.

Reply:
FYI `./hack/test-all.sh` did not pass for me when I pulled and ran it locally

Reply:
oops, this was a bad merge